app/controllers/api/v1/sessions_controller.rb

```diff
@@ -43,6 +43,11 @@ def session_params
       def sign_in(user)
         session[:user_id] = user.id
       end
+
+      # Signs Out the user
+      def sign_out
+        session[:user_id] = nil
+      end
```

Comment:
No need to break this off into its own function

Reply:
Done